src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -99,18 +95,23 @@ public DefaultTokenCacheStore(Context context) {
             throw new IllegalArgumentException("Context is null");
```

Comment:
nit: move it to the first line if null throw right away. Just simpler to read code